packages/backend/src/postgres.ts

```diff
@@ -82,6 +82,7 @@ import { MiReversiGame } from '@/models/ReversiGame.js';
 import { Config } from '@/config.js';
 import MisskeyLogger from '@/logger.js';
 import { bindThis } from '@/decorators.js';
+import { MiNoteDraft } from './models/NoteDraft.js';
```

Comment:
```suggestion
import { MiNoteDraft } from '@/models/NoteDraft.js';
```

にしたうえで他のmodelsと同列に置き直したほうが良さそう